omero/users/cli/containers-annotations.txt

```diff
@@ -0,0 +1,63 @@
+Creating containers and annotations
+-----------------------------------
+
+The :omerocmd:`obj` command allows users to create and update OMERO objects,
+this can be used to create containers, i.e. projects, datasets and screens. It
```

Comment:
The page doesn't mention import-into-container and we create containers after import too. `obj` works fine for creating folders and could be quite useful for, e.g., people scripting getting an ontology into OMERO, but I don't strongly object to your omitting them: this point is not a blocker.